src/staticanalysis/bot/static_analysis_bot/clang/__init__.py

```diff
@@ -18,30 +24,35 @@ def setup(
     artifact='public/build/clang-tidy.tar.xz',
     repository='autoland',
 ):
-    '''
```

Comment:
It would be cleaner to put all the initial code in a function in the same file (for example `download_artifacts`), and then call the function in the Exception case.

Also the diff would be more readable